README.md

```diff
@@ -1,12 +1,38 @@
+# GWFish
+
+## About
+
 <p align="center">
   <img src="gwfish-1.png" width="200" title="Logo">
 </p>
-Simulation of detector networks with Fisher-matrix PE
+Simulation of gravitational-wave detector networks with Fisher-matrix PE
 
 Shield: [![CC BY 4.0][cc-by-shield]][cc-by]
 
 Read the documentation [here](https://gwfish.readthedocs.io)!
 
+## Citation
```

Comment:
Adding the citation is a great idea!
However, I think this is formatted in a more complicated way than it needs: you can just use a code block, like


```
@ARTICLE{DupletsaHarms2023,
        author = {{Dupletsa}, U. and {Harms}, J. and {Banerjee}, B. and {Branchesi}, M. and {Goncharov}, B. and {Maselli}, A. and {Oliveira}, A.~C.~S. and {Ronchini}, S. and {Tissino}, J.},
        title = "{GWFISH: A simulation software to evaluate parameter-estimation capabilities of gravitational-wave detector networks}",
        journal = {Astronomy and Computing},
        keywords = {General Relativity and Quantum Cosmology},
        year = 2023,
        month = jan,
        volume = {42},
        eid = {100671},
        pages = {100671},
        doi = {10.1016/j.ascom.2022.100671},
        archivePrefix = {arXiv},
        eprint = {2205.02499},
        primaryClass = {gr-qc},
        adsurl = {https://ui.adsabs.harvard.edu/abs/2023A&C....4200671D},
        adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```

(so, using three backticks: \`\`\` at the beginning and end)

Reply:
and in there you can just have regular spaces, regular newlines and so forth (it's all verbatim rendered)

Reply:
Thanks, done!